content/articles/intro-to-emberjs/index.md

```diff
@@ -0,0 +1,155 @@
+
+
+
+This creates an application development environment that is ready for production and includes the following features by default:
+
+* 	Live reloading development server.
+* 	Third-party packages can richly enrich your application thanks to the plugin architecture.
+* 	Babel and Webpack integration for the newest JavaScript.
+* 	Automated testing environment that allows you to test like a user by running your tests in the browser.
+* 	CSS and JavaScript transpilation and minification for a production build.
+* 	The use of conventions to reduce the variations across applications (allowing easier mental context switching).
+	
+### Installing the shared assets for TodoMVC projects
+
+Installing shared assets, like we're going to do, isn't usually a compulsory step for new projects, but it does allow us to leverage existing shared CSS, which eliminates the need to guess what CSS is required to construct the TodoMVC styles.
+1.	To begin, open a terminal window and navigate to your `todomvc` directory, for example, cd todomvc in macOS/Linux
+
+2.	To add the common todomvc CSS to your app, use the following command
+
+           npm install --save-dev todomvc-app-css todomvc-common
+
+3.	Next, open the `ember-cli-build.js` file in your preferred code editor inside the todomvc directory (it's right there inside the root).
+
+4.	In ember-cli-build.js, find the following code:
+
+         let app = new EmberApp(defaults, {
+          // Add options here
+         });
+
+5.	Before saving the file, add the following lines underneath it:
+
+
+         app.import('node_modules/todomvc-common/base.css');
+         app.import('node_modules/todomvc-app-css/index.css');
+
+6.	Finally, find `app.css`, located at `app/styles/app.css`, and paste in the following:
+
+
+
+          :focus,
+         .view label:focus,
+         .todo-list li .toggle:focus + label,
+         .toggle-all:focus + label {
+          /* ! Important since the outline is intentionally disabled in todomvc styles. */
+          outline: #d86f95 solid !important;
+          }
+
+This CSS overrides some of the styles supplied by the `todomvc-app-css` npm package, making it possible to see the keyboard focus. This goes a long way toward addressing one of the TodoMVC app's key accessibility issues.
+
+Starting the development server
+In your terminal, type the following command while within the `todomvc` directory to launch the app in development mode:
+
+        ember server
+
+The development server is available at `http://localhost:4200`, which you can use to see how far your work has progressed.
+
+###  Difference between React JS, Angular JS,  Ember JS, and Vue JS
+
+| Attributes | React JS | Angular JS | Ember JS | Vue JS |
+| :---: | :---: | :---: | :---: | :---: |
+| **type** | Library to build UI | A framework | A framework | A framework |
+|**Why choose?**|	If you like all things JavaScript | If you love TypeScript | If you like all things JavaScript | Easy JavaScript & HTML |
```

Comment:
This may be a reason. But, the choice behind each tech stack is entirely different. Please find that out. You can't generalize stuff like these. It needs proper backed research.